app/javascript/controllers/expandcollapse_controller.js

```diff
@@ -0,0 +1,29 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static outlets = ["description"]
+  static targets = ["expandButton", "collapseButton"]
+
+  expandDescriptions() {
+    this.descriptionOutletElements.forEach(function (element) {
```

Comment:
I think this is way better! However, I'd use `descriptionOutlets` and call `expand` on each controller.  Then the description controller can decide how to implement expand (in this case it will remove the `collapse` class.

Reply:
Nice, thanks. Done!